pilot/lib/pilot/build_manager.rb

```diff
@@ -127,6 +133,16 @@ def wait_for_processing_build
           builds = app.all_processing_builds
           break if builds.count == 0
           latest_build = builds.last
+
+          if latest_build.valid and must_update_build_info
+            # Set the changelog and/or description if necessary
+            if should_update_build_information(options)
+              build.update_build_information!(whats_new: options[:changelog], description: options[:beta_app_description], feedback_email: options[:beta_app_feedback_email])
+              UI.success "Successfully set the changelog and/or description for build"
+            end
+            must_update_build_info = false
```

Comment:
I'm not sure I understand. I need to set this flag in order **not** to update build info on every pass of the loop while still waiting for the build to be fully processed. Am I missing something?